src/tests/dispatch-payload.test.ts

```diff
@@ -116,11 +117,18 @@ Deno.test("DispatchPayload function file compatibility tests", async (t) => {
   const fixturesDir = `${__dirname}/fixtures`;
   Deno.chdir(fixturesDir);
   const functionsDir = `${fixturesDir}/functions`;
+  mockFetch.install(); // mock out calls to fetch
 
   await t.step(
     "return from provided file",
```

Comment:
This test, and the test below, seems extraneous? I am not sure the intention of these two tests, and they seem to cover functionality that is already tested elsewhere.
I vote we remove this particular `Deno.test` block.

Reply:
In LoadFunction there is, but not in the context of DispatchPayload. I will keep the file-not-found test but remove the other one.